src/Target.h

```diff
@@ -50,6 +50,24 @@ struct Target {
     /** The bit-width of the target machine. Must be 0 for unknown, or 32 or 64. */
     int bits = 0;
 
+    /** The specific processor to be targeted, tuned for.
```

Comment:
I would like a comment after each processor giving a longer name, such as the manufacturer and "Zen" expanded, possibly the year they were introduced, etc. These may make sense to the person who wrote them today, but they will not a year from now or necessarily to anyone else. Basically enough to give one an idea if the chip one has handy matches something in the list and to help Google searches narrow it down.

Reply:
Should be done, do tell if you want it to be even more detailed.